src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -949,4 +949,73 @@ Configuration is disabled due to Group Policy.</value>
   <data name="WindowsLibrariesDependencies" xml:space="preserve">
     <value>Windows Libraries</value>
   </data>
+  <data name="WindowsStoreTerms" xml:space="preserve">
+    <value>Windows Store Terms</value>
+  </data>
+  <data name="AcceptAgreementsArgumentDescription" xml:space="preserve">
+    <value>Accept all license agreements</value>
+  </data>
+  <data name="ExportedPackageRequiresLicenseAgreement" xml:space="preserve">
+    <value>Exported package requires license agreement to install:</value>
+  </data>
+  <data name="LicenseAgreementPrompt" xml:space="preserve">
+    <value>The publisher requires that you view the following information and accept the EULA before installing.
+Do you agree to the terms?</value>
+  </data>
+  <data name="LicenseNotAgreedTo" xml:space="preserve">
+    <value>License not agreed to. Installation cancelled.</value>
+  </data>
+  <data name="ShowLabelAgreements" xml:space="preserve">
+    <value>Agreements:</value>
+  </data>
+  <data name="ShowLabelAuthor" xml:space="preserve">
+    <value>Author:</value>
+  </data>
+  <data name="ShowLabelDescription" xml:space="preserve">
+    <value>Description:</value>
+  </data>
+  <data name="ShowLabelInstaller" xml:space="preserve">
+    <value>Installer:</value>
+  </data>
+  <data name="ShowLabelInstallerLocale" xml:space="preserve">
+    <value>Locale:</value>
+  </data>
+  <data name="ShowLabelInstallerProductId" xml:space="preserve">
+    <value>Store Product Id:</value>
+  </data>
+  <data name="ShowLabelInstallerSha256" xml:space="preserve">
+    <value>SHA256:</value>
+  </data>
+  <data name="ShowLabelInstallerType" xml:space="preserve">
+    <value>Type:</value>
+  </data>
+  <data name="ShowLabelInstallerUrl" xml:space="preserve">
+    <value>Download Url:</value>
+  </data>
+  <data name="ShowLabelLicense" xml:space="preserve">
+    <value>License:</value>
+  </data>
+  <data name="ShowLabelLicenseUrl" xml:space="preserve">
+    <value>License Url:</value>
+  </data>
+  <data name="ShowLabelMoniker" xml:space="preserve">
+    <value>Moniker:</value>
+  </data>
+  <data name="ShowLabelPackageUrl" xml:space="preserve">
+    <value>Homepage:</value>
+  </data>
+  <data name="ShowLabelPublisher" xml:space="preserve">
+    <value>Publisher:</value>
+  </data>
+  <data name="ShowLabelVersion" xml:space="preserve">
+    <value>Version:</value>
+  </data>
+  <data name="PromptOptionNo" xml:space="preserve">
+    <value>&amp;No</value>

```

Comment:
>&amp; [](http://example.com/codeflow?start=11&length=5)

nit: This part should be output by our code, it's not part of the localization string..

Reply:
I tried to copy what PS does for localization when it does prompts like this. The idea is that the hotkey used could be different in each localization, e.g. "[Y]es" in English but "[S]i" in Spanish. Their solution (that I copied) is to have that '&' in the localized string to signal which character is the hotkey, because it may not be the first one if Yes and No start the same.

Reply:
Sorry I did not read the comments. If the loc team understands this, then it's perfectly fine.